ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_core.cc

```diff
@@ -326,8 +326,7 @@ TFCore::getOutputTensorDim (GstTensorsInfo * info)
 }
 
 #define copyInputWithType(type) do { \
-    for (int idx = 0; idx < array_len; ++idx) \
-      inputTensor.flat<type> ()(idx) = ((type*) input[i].data)[idx]; \
+    memcpy (inputTensor.flat<type>().data(), input[i].data, input[i].size); \
```

Comment:
tested with object detection example

before using memcpy (loop)
** Message: Invoke() is finished: 770173
** Message: Invoke() is finished: 757116
** Message: Invoke() is finished: 765512

memcpy for input data
** Message: Invoke() is finished: 669438
** Message: Invoke() is finished: 661400
** Message: Invoke() is finished: 685049